ruby/ruby_releases/ruby_benchmarks/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Build base image for Ruby benchmarks
+cd ../ruby_releases_base
+sudo docker build -t rubybench/ruby_releases_base:0.4 .
+cd ../ruby_benchmarks
+sudo docker build -t rubybench/ruby_releases .
```

Comment:
That isn't how it works. 

Basically the scripts are used so that I don't have to type `docker build -t rubybench/ruby_releases` everytime. The version is still controlled by the  user. That means the script for ruby_releases_base needs to accept a variable for the tag